cmd/sfncli/sfncli.go

```diff
@@ -33,10 +34,22 @@ func main() {
 		os.Exit(0)
 	}
 
-	if *name == "" {
-		fmt.Println("name is required")
+	if *activityName == "" {
+		fmt.Println("activityname is required")
 		os.Exit(1)
 	}
+	*activityName = os.ExpandEnv(*activityName)
+	if *workerName == "" {
+		fmt.Println("workername is required")
+		os.Exit(1)
+	}
+	*workerName = os.ExpandEnv(*workerName)
+	if newWorkerName, err := expandECSTaskARN(*workerName); err != nil {
+		fmt.Printf("error expanding %s: %s", magicECSTaskARN, err)
+		os.Exit(1)
+	} else {
```

Comment:
we don't really need this else but good either way :)

Reply:
it restricts the scope of `newWorkerName` to these 7 lines of code :stuck_out_tongue_winking_eye: